lib/translator/inquiry.cc

```diff
@@ -158,6 +158,36 @@ void TranslateUnitSerialNumberVpd(
   WriteValue(product_serial_number, buffer.subspan(sizeof(result)));
 }
 
+StatusCode TranslateBlockDeviceCharacteristicsVpd(Span<uint8_t> buffer) {
+  scsi::BlockDeviceCharacteristicsVpd block_device_characteristics_vpd = {
+
+      // Shall be set to 00h indicating direct access block device
+      .peripheral_device_type = scsi::PeripheralDeviceType::kDirectAccessBlock,
+
+      // Shall be set to 000b indicating device for PERIPHERAL DEVICE TYPE
+      // connected to logical unit
+      .peripheral_qualifier =
+          scsi::PeripheralQualifier::kPeripheralDeviceConnected,
+
+      // Shall be set to B1h indicating Device Identification VPD Page
+      .page_code = scsi::PageCode::kDeviceIdentification,
+
+      // Shall be set to 3Ch.
+      .page_length = scsi::PageLength::kBlockDeviceCharacteristicsVpd,
+
+      // Shall be set to 0001h indicating a non-rotating device(SSD)
+      .medium_rotation_rate = scsi::MediumRotationRate::kNonRotatingMedium,
+
+      // Shall be set to 0h indicating form factor not reported
+      .nominal_form_factor = scsi::NominalFormFactor::kNotReported};
+
+  if (!WriteValue(block_device_characteristics_vpd, buffer)) {
+    DebugLog("Error! Couldn't write BlockDeviceCharacteristicsVpd to buffer\n");
```

Comment:
```suggestion
    DebugLog("Couldn't write BlockDeviceCharacteristicsVpd to buffer\n");
```
The error is implied.